pkg/controller/clusterinstallation/minio.go

```diff
@@ -28,6 +28,12 @@ func (r *ReconcileClusterInstallation) checkMinio(mattermost *mattermostv1alpha1
 }
 
 func (r *ReconcileClusterInstallation) checkMinioSecret(mattermost *mattermostv1alpha1.ClusterInstallation, reqLogger logr.Logger) error {
+	// Check if custom secret was specified
+	if mattermost.Spec.Minio.Secret != "" {
+		reqLogger.Info("Skipping minio secret creation, using custom secret")
+		return nil
```

Comment:
Should I also fetch the secret to check if it really exists?

Reply:
yes, will be good to validate if exist and if that contains the required fields

Reply:
@cpanato I added the check. Thanks